src/plugins/physics/AmmoPhysics.js

```diff
@@ -0,0 +1,42 @@
+define(["lib/classy",
+        "core/framework/TundraSDK",
+        "core/framework/ITundraPlugin",
+        "core/framework/TundraLogging",
+        "core/scene/Scene",
+        "plugins/physics/PhysicsWorld_Ammo",
+        "plugins/physics/EC_RigidBody_Ammo"
+    ], function(Class, TundraSDK, ITundraPlugin, TundraLogging, Scene, PhysicsWorld_Ammo, EC_RigidBody_Ammo) {
+/**
+    This plugin provides client side ammo physics.
+
+    @class AmmoPhysics
+    @extends ITundraPlugin
+    @constructor
+*/
+var AmmoPhysics = ITundraPlugin.$extend(
+{
+    __init__ : function(targetScene)
+    {
+        this.$super("AmmoPhysics");
+        
+        this.log = TundraLogging.getLogger("AmmoPhysics");
```

Comment:
ITundraPlugin takes care of the logger with the plugin name. This line can be removed.

https://github.com/realXtend/WebTundra/blob/dev2/src/core/framework/ITundraPlugin.js#L38